### PR TITLE
LHC-482 tweak label-sm spacing

### PR DESCRIPTION
**Pull request by rdai10**
### Description
Adjust label spacing per designer suggestion.

### CODE REVIEW

File: theme/src/css/_zendesk-elements.scss [hunk 1]
```diff
@@ -143,6 +143,7 @@ textarea,
 
 .label-sm {
 	@include clay-label-size($label-sm);
```
Comment (jonmak08) [inline theme/src/css/_zendesk-elements.scss]: Add a line break between mixin and CSS property.  See https://github.com/rdai10/zendesk/blob/123c9b5f26613df574cd25ba2edeaf4e5a07e7cd/theme/src/css/_zendesk-notification.scss#L2...L4 for an example.

Comment (rdai10) [reply theme/src/css/_zendesk-elements.scss]: Will do!

Comment (rdai10) [reply theme/src/css/_zendesk-elements.scss]: https://github.com/rdai10/zendesk/pull/71

Comment (jonmak08) [reply theme/src/css/_zendesk-elements.scss]: Thanks @rdai10!

**Pull request merged**